surfsense_backend/app/services/connector_service.py

```
@@ -539,78 +598,91 @@ async def search_extension(self, user_query: str, user_id: str, search_space_id:
-                webpage_url = metadata.get('VisitedWebPageURL', '')
-                visit_date = metadata.get('VisitedWebPageDateWithTimeInISOString', '')
-                visit_duration = metadata.get('VisitedWebPageVisitDurationInMilliseconds', '')
-                browsing_session_id = metadata.get('BrowsingSessionId', '')
-                
+                webpage_title = metadata.get("VisitedWebPageTitle", "Untitled Page")
+                webpage_url = metadata.get("VisitedWebPageURL", "")
+                visit_date = metadata.get("VisitedWebPageDateWithTimeInISOString", "")
+                visit_duration = metadata.get(
+                    "VisitedWebPageVisitDurationInMilliseconds", ""
+                )
+                browsing_session_id = metadata.get("BrowsingSessionId", "")
```

Ruff (0.12.2)

Local variable `browsing_session_id` is assigned to but never used

Remove assignment to unused variable `browsing_session_id`

(F841)